frontend/public/components/overview/pods-overview.tsx

```diff
@@ -173,22 +181,122 @@ export const PodsOverview: React.SFC<PodsOverviewProps> = ({
         </Alert>
       ) : null}
       {_.isEmpty(filteredPods) ? (
-        <span className="text-muted">{emptyMessage}</span>
+        <span className="text-muted">{loaded ? loadError || emptyMessage : <LoadingBox />}</span>
```

Comment:
Hmm... looks like if we get a gateway error because the cluster going down, this doesn't really render properly; it's stuck not loaded but with an error. Did we want to check loaded and NOT an error?

React Dev Tools & the Response (Error) Object:
![Screen Shot 2020-11-03 at 7 10 27 PM](https://user-images.githubusercontent.com/8126518/98054207-58018b80-1e08-11eb-95c1-0115421ec45e.png)
![Screen Shot 2020-11-03 at 7 10 34 PM](https://user-images.githubusercontent.com/8126518/98054208-58018b80-1e08-11eb-8116-f686c2e8b46a.png)